### ISSUE TITLE
Disable configuring the Management0 interface and the management VRF on leafs with inband mgmt

### ISSUE THREAD

**Issue by carlbuchmann** [aristanetworks/ansible-avd#3400]
### Discussed in https://github.com/aristanetworks/ansible-avd/discussions/3397

<div type='discussions-op-text'>

<sup>Originally posted by **kmueller68** December  1, 2023</sup>
Hello.

In the AVD campus fabric example is exmplained how to disable generation of oob interfaces on leafs.

In Campus Networks, having a dedicated out-of-band management network in each IDF is uncommon. Therefore, you can easily disable configuring the Management0 interface and the management VRF by adding these variables to the DC1_LEAFS.yml group_vars.
```
mgmt_gateway: null
mgmt_interface: null
mgmt_interface_vrf: default
```

Should this really work?
I have tested it but the `interface Management1`is still created including an IP?

```
---
### group_vars/DC1_LEAFS.yml

type: leaf     # Must be leaf
mgmt_gateway: null
mgmt_interface: null
mgmt_interface_vrf: default
```

thank you
Karlheinz
</div>

**Comment by CyrielRct** [comment]
Try to define your leaf by inband_mgmt_vlan  instead of mgmt_interfaces : 

mgmt_interface: Vlan100 << specify your inband mgmt interface
mgmt_interface_vrf: default

leaf:
  defaults:
    platform: cEOSLab
    inband_mgmt_vlan: 100
    inband_mgmt_gateway: 10.x.x.x

**Comment by ClausHolbechArista** [comment]
To the implementer: I think we have solved this by now, but please verify.

**Comment by laxmikantchintakindi** [comment]
Yes. This issue is resolved. We are not generating Mgmt interface or VRF when using inband_mgmt.

**Comment by laxmikantchintakindi** [comment]
But we are generating this config for inband_mgmt configuration.

```
management api http-commands
   protocol https
   no shutdown
   !
   vrf MGMT
      no shutdown
```
I think this will be solved in 6.0 when we use default `use_default_mgmt_method_vrf`.

**Comment by laxmikantchintakindi** [comment]
Closing the issue as problem is solved in now.
